packages/api/cms-api/src/mikro-orm/migrations/Migration20240702123233.ts

```diff
@@ -0,0 +1,43 @@
+import { Migration } from "@mikro-orm/migrations";
+import * as mimedb from "mime-db";
+import { getValidExtensionsForMimetype } from "../../dam/files/files.utils";
```

Comment:
We normally recommend to not add application code to migrations, since old migrations need to be updated if the code changes. Should we duplicate the function here or make an exception?

Reply:
Makes sense. It's not that much code anyway so I copied it: 3a943089ad5263015abc8aef41b1350fe2b86fe8

We still have the dependency on mimedb though, but I don't think that's a problem